site/docs/getting-started.mdx

```diff
@@ -4,8 +4,102 @@ sidebar_position: 1
 
 # Getting Started
 
-Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+## Developing
 
-## Lorem ipsum
+Learn how to use Odyssey's [UI components](/components/) in your React web applications.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+### Pre-requisites
+
+In order to use our components you will need:
+
+- Familiarity with web development fundamentals (HTML, CSS & JavaScript) and [React](https://reactjs.org/)
+- [Node.js](https://nodejs.org/) v16 or later installed on your system.
+- A React (v16 or later) web application project.
+  - For new projects, you can use [`create-react-app`](https://github.com/facebook/create-react-app#creating-an-app) or similar tools.
+
+### Install packages
+
+You will need to add the following three dependencies to your project:
+
+```sh
+npm install --save @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons
+```
+
+:::note
+
+If you prefer, you can use [`yarn`](https://yarnpkg.com/) or other package managers to install our NPM packages.
+
+:::
+
+- `@jpmorganchase/uitk-core` contains our production-ready UI components
+- `@jpmorganchase/uitk-theme` contains the CSS files that apply Odyssey's default theme
+- `@jpmorganchase/uitk-icons` contains our SVG-based icons
+
+:::tip
+
+We strongly recommend that you regularly update to newer, compatible versions of Odyssey as they are released. For example, if using `npm`, this can be achieved by running:
+
+```sh
+npm update @jpmorganchase/uitk-core @jpmorganchase/uitk-theme @jpmorganchase/uitk-icons
+```
+
+This will provide you with the latest bug fixes and enhancements and does not require you to update any of your own code.
```

Comment:
Only true after a 1.0.0 release

Reply:
Ah yes. Good point!

However, having just re-read the SemVer docs, shouldn't our December release be 1.0.0? The component APIs we'll be shipping as part of that are meant to be stable, no?

Reply:
I think we still need to confirm that.

Reply:
OK. I'll just remove this "tip" section for the time being as it's not essential.

Will add something like that back in whenever we do ship a v1.0.0